.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[{package,bower}.json]
```

Comment:
@battaglr Can you apply this rule to all the json files please? Like a `[*.json]`.